org-formation/organization.yaml

```diff
@@ -425,12 +425,12 @@ Organization:
     Type: OC::ORG::Account
     Properties:
       AccountName: org-sagebase-dnt-dev
-      RootEmail: ibc-ops@sagebase.org
+      RootEmail: aws-dpe-dev@sagebase.org
```

Comment:
the `ibc-ops@sagebase.org` google group still exists.  do you plan on removing it?

Reply:
IBC is no longer a team or group at Sage, so we have been sunsetting the usage of that group.